### PR TITLE
Add v0 of HWP angle loading

### PR DESCRIPTION
**Pull request by kmharrington**
Expand the g3tsmurf loading from configs a bit and add a utility function for loading up HWP angles. This assumes the HWP angles are saved in the format generated by the `update-hwp-angles` site pipeline script. 

The loading for the HWP data is _slow_. We're going to have to fix that soon-ish. But this does work and does get the HWP angles loading into the AxisManager. We can upgrade this function when we get faster methods.

### CODE REVIEW

File: sotodlib/io/g3tsmurf_utils.py
Comment (mhasself) [inline sotodlib/io/g3tsmurf_utils.py:242]: This sort of thing doesn't work on data with a branch cut, right?

A brute force solution is to "unwrap" the HWP angle and resample that, then put the result back in the [0, 2pi) branch.  float64 should have enough precision to handle that without adding noise, at least for ~days of data at a time.

File: sotodlib/io/g3tsmurf_utils.py
Comment (msilvafe) [inline sotodlib/io/g3tsmurf_utils.py:216]: Can you point to an example `hwp_prefix` file or dictionary in the doc-string or readthedocs?

**Review by msilvafe — CHANGES_REQUESTED** [review]
The main actual comment is just addressing what Matthew brought up. I think if you address that and have tested that it works this can be merged. 

Otherwise I think it would be useful if you included in the PR comment, doc string, or readthedocs  an example of what the loading in the presence of HWP data looks like (like what's the function call) and what is the structure of the config file.

File: sotodlib/io/g3tsmurf_utils.py
Comment (kmharrington) [reply sotodlib/io/g3tsmurf_utils.py:242]: @msilvafe  any chance you have the code for that you could just paste into this and merge into this branch? I'm more interested in connecting all the IO dots and would rather leave the math to the SAT folks.

File: sotodlib/io/load_smurf.py
Comment (kmharrington) [reply sotodlib/io/load_smurf.py:202]: https://simons1.princeton.edu/confluence/display/CHILAT/SO%3A+Data+Packaging+FLP#SO:DataPackagingFLP-P10R1DataLoadingExample

File: sotodlib/io/g3tsmurf_utils.py
Comment (kmharrington) [reply sotodlib/io/g3tsmurf_utils.py:216]: Actually, I guess you also haven't seen these changes? https://sotodlib.readthedocs.io/en/latest/hwp.html

File: sotodlib/io/g3tsmurf_utils.py
Comment (msilvafe) [inline sotodlib/io/g3tsmurf_utils.py:49]: Also this logger is trying to return `obs.obs_id` but `obs` isn't defined.

File: sotodlib/io/g3tsmurf_utils.py
Comment (msilvafe) [inline sotodlib/io/g3tsmurf_utils.py:66]: Same here as line 49.

**Review by msilvafe — APPROVED** [review]
I'm going to go ahead and approve so that I'm not blocking merging but I would like to see this tested with the commits that I added in before it gets merged.

**Comment by kmharrington** [comment]
looks good, and I've testing it again. Thanks Max

**Pull request merged**
